forge/csrc/passes/limit_to_5d_reshape.cpp

```diff
@@ -20,7 +20,7 @@ static bool is_reshape(graphlib::Node const *node)
     return op and op->op_name() == "reshape";
 }
 
-void limit_to_4d_reshape(graphlib::Graph *graph)
+void limit_to_5d_reshape(graphlib::Graph *graph)
```

Comment:
Please add function documentation here. Just to have a wider context what it does, and why.

Reply:
Done